agent-core/src/main/java/com/newrelic/agent/android/logging/LogReporting.java

```diff
@@ -163,13 +162,17 @@ public void log(LogLevel logLevel, String message) {
         }
 
         public void logThrowable(LogLevel logLevel, String message, Throwable throwable) {
-            StringWriter sw = new StringWriter();
 
+            if (null != throwable) {
+                StringWriter sw = new StringWriter();
+                throwable.printStackTrace(new PrintWriter(sw));
+                message = String.format(Locale.getDefault(), "%s: %s", message, sw);
```

Comment:
Don't replace message with a non-null throwable. Both may be valid. Your condition should only be if throwable is null.

Reply:
changes done.